sql/sql_class.h

```diff
@@ -3323,6 +3323,13 @@ class THD : public MDL_context_owner,
   Gtid_set owned_gtid_set;
 #endif
 
+#ifdef HAVE_OPENSSL
```

Comment:
Should `connection_certificate()` / `set_connection_certificate()` also be wrapped with `#ifdef HAVE_OPENSSL`?
Because otherwise, this does not seem to compile without `HAVE_OPENSSL` defined

Reply:
That's a trivial setter / getter, works without SSL.

Reply:
Actually, my main concern here was that when `HAVE_OPENSSL` is not defined, the class won't have `m_connection_certificate` member and therefore `connection_certificate() / set_connection_certificate()` will fail to compile as `m_connection_certificate = cert;` / `return m_connection_certificate;` will be an invalid code.

Reply:
Fixed.